docker/Dockerfile

```diff
@@ -142,27 +82,21 @@ COPY --from=stage_build /emsdk /emsdk
 # Fallback in case Emscripten isn't activated.
 # This will let use tools offered by this image inside other Docker images (sub-stages) or with custom / no entrypoint
 ENV EMSDK /emsdk
-ENV EMSCRIPTEN=${EMSDK}/emscripten/sdk
-
-ENV EM_DATA ${EMSDK}/.data
-ENV EM_CONFIG ${EMSDK}/.emscripten
-ENV EM_CACHE ${EM_DATA}/cache
-ENV EM_PORTS ${EM_DATA}/ports
-
-# Fallback in case Emscripten isn't activated
-# Expose Major tools to system PATH, so that emcc, node, asm2wasm etc can be used without activation
-ENV PATH="${EMSDK}:${EMSDK}/emscripten/sdk:${EMSDK}/llvm/clang/bin:${EMSDK}/node/current/bin:${EMSDK}/binaryen/bin:${PATH}"
 
 # Use entrypoint that's coming from emscripten-slim image. It sets all required system paths and variables
 ENTRYPOINT ["/emsdk/entrypoint"]
 
 
 # ------------------------------------------------------------------------------
 # Create a 'standard` 1000:1000 user
-# Thanks to that this image can be executed as non-root user and created files will not require root access level on host machine
-# Please note that this solution even if widely spread (i.e. Node.js uses it) is far from perfect as user 1000:1000 might not exist on
-# host machine, and in this case running any docker image will cause other random problems (mostly due `$HOME` pointing to `/`)
-# This extra user works nicely with entrypoint provided in `/emsdk/entrypoint` as it detects case explained before.
+# Thanks to that this image can be executed as non-root user and created files
+# will not require root access level on host machine Please note that this
+# solution even if widely spread (i.e. Node.js uses it) is far from perfect as
+# user 1000:1000 might not exist on host machine, and in this case running any
+# docker image will cause other random problems (mostly due `$HOME` pointing to
+# `/`)
+# This extra user works nicely with entrypoint provided in `/emsdk/entrypoint`
+# as it detects case explained before.
```

Comment:
This over-complicated entrypoint was created to support running the image as non-root user. (We've found that just easier in some Jenkins pipelines) so that non-root user can wipe out created files from mounted volume without having `sudo`. 
I believe that once `chmod -R 777 ${EMSDK}` was performed, the folder structure wasn't fully created. Like I remember that at that moment no ports were downloaded (or cache for them). 
Then trying to compile using the image will fail as no access to write to a folder. 

That leads to the point of taking a decision if you would like to support this scenario.

Reply:
Ah I see.   

I think we can continue to support that use case.   I've added "umask 777" to /root/.bashrc and I think we can just remove the special entrypoint.

See what you think.